pkgs/development/python-modules/python-lsp-ruff/default.nix

```diff
@@ -32,19 +32,19 @@ buildPythonPackage rec {
     sed -i -e "s|workspace.root_path|'/tmp/'|g" tests/*.py
   '';
 
-  propagatedBuildInputs = [
+  dependencies = [
```

Comment:
Is this the new style for python packages?

Reply:
Yes ! This has been the new standard for a few months now.